prow/gerrit/adapter/adapter.go

```diff
@@ -397,6 +398,9 @@ func (c *Controller) processChange(logger logrus.FieldLogger, instance string, c
 			}
 		}
 		if err != nil {
+			if err := c.gc.SetReview(instance, change.ID, change.CurrentRevision, failedIRCMsg, nil); err != nil {
+				return err
```

Comment:
return error without any wrapper will appear in log like failed commenting on PR, while in this case the real error was inrepoconfig failure. suggest return both errors like `if setReviewErr := c.gc.SetReview(...); setReviewErr != nil {return fmt.Errorf("something something", setReviewErr, err)}`

Reply:
Done